datadog_lambda/handler.py

```diff
@@ -0,0 +1,28 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2020 Datadog, Inc.
+
+from __future__ import absolute_import
+from importlib import import_module
+
+import os
+from datadog_lambda.wrapper import datadog_lambda_wrapper
+
+
+class HandlerError(Exception):
+    pass
+
+
+path = os.environ.get("DATADOG_USER_HANDLER", None)
```

Comment:
Let's be consistent with other environment variable and use prefix `DD_`? Also, how about `DD_LAMBDA_HANDLER`?